stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/MessageListView.kt

```diff
@@ -515,6 +517,7 @@ public class MessageListView : ConstraintLayout {
         messageOptionsConfiguration = MessageOptionsView.Configuration(
             iconsTint = iconsTint,
             replyIcon = replyIcon,
+            replyEnabled = replyEnabled,
```

Comment:
Maybe all "enabled" attributes at the bottom? Or we can move the "copyTextEnabled" field above "copyIcon" later.